### PR TITLE
Rename project to heroku-data-privatelink-cli

### PR DESCRIPTION
**Pull request by jdowning**
This commit expands the use of the privatelink cli to incorporate all
data addon types. This involves renaming a lot of `pg` specific commands
and references to now be prefixed with `data`. As such, the package name
has also been updating as well as the version to denote the major
change.

All previous `pg` prefixed commands are aliased to their new `data`
prefixed commands.

### CODE REVIEW

**Comment by brettgoulder** [comment]
Awesome! I do have one concern, what to do about existing customers who have the plugin installed already. I do not think that the CLI's plugin system will pick this change up since we'd be doing a package change on npmjs as well, so we'd likely need to do outreach.  

@RasPhilCo have you ever needed to rename a plugin before?

**Comment by RasPhilCo** [comment]
@brettgoulder Yep!
1) Publish this package to npm
2) add a PR to CLI that adds a plugin update migration (ex: https://github.com/heroku/cli/blob/master/packages/cli/package.json#L251)

**Comment by RasPhilCo** [comment]
Whoops! 😬

**Comment by brettgoulder** [comment]
Even better! Y'all thought of everything :D

File: package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@heroku-cli/plugin-pg-privatelink",
-  "description": "Heroku Postgres via PrivateLink CLI",
-  "version": "0.8.0",
+  "name": "@heroku-cli/plugin-data-privatelink",
+  "description": "Heroku Data via PrivateLink CLI",
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (jdowning) [reply package.json]: @brettgoulder I can amend. What would you like to do here?

[the remark this answers is not in the document]
Comment (brettgoulder) [reply package.json]: @jdowning keep it at 0.8.0

[the remark this answers is not in the document]
Comment (jdowning) [reply package.json]: @brettgoulder done 👍

**Pull request merged**